id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/interceptor/IdRepoEntityInterceptor.java

```diff
@@ -130,7 +132,7 @@ private <T extends UinInfo> void encryptDataOnSave(Serializable id, Object[] sta
 	public boolean onLoad(Object entity, Serializable id, Object[] state, String[] propertyNames, Type[] types) {
 		try {
 			List<String> propertyNamesList = Arrays.asList(propertyNames);
-			if (entity instanceof Uin || entity instanceof UinHistory || entity instanceof UinDraft) {
+			if (entity instanceof Uin || entity instanceof UinDraft) {
```

Comment:
Please check in this if it is present i release branch, add it back.

Reply:
Done